product/publishing.mdx

```diff
@@ -86,3 +86,34 @@ Each version will provide information on what was published and when. You can gi
 <Frame>![Publish history](/images/builder/publishing5.png)</Frame>
 
 To bring your live site back to the latest version, you can either publish the most recent version in your History tab, or publish new changes to your site in the Changes tab.
+
+## Best Practices
+
+Maintaining a consistent publishing workflow is key to keeping your site in sync with your builder. Here are some best practices to follow:
+
+### Utilize Previews
+
+Previews are a great way to test your changes before publishing them. You can find the preview button at the top right of the builder's browser bar.
+
+<Frame>![Preview button](/images/builder/publishing1.png)</Frame>
+
+Every workspace comes with unlimited [Guests](/product/workspace/user#guests) who can view your site in preview mode.
+
+### Publish dialog hygiene
+
+You should strive to keep your publish dialog clean and organized. This will help you avoid accidentally publishing changes you haven't reviewed or want to publish. To do this, you should:
+
+- Aim to have your publish dialog clean before starting a new session.
+- Be aware of what is in the dialog ahead of editing or creating a new page, resource, or component.
+- If you're working on a team, communicate with your team members about what you are working on and what you are publishing.
+
+### Publishing in batches
+
+Publishing in batches is a great way to keep track of the state of your site in different points in time. This will also help you leverage the site history feature to revert to a previous state of your site.
```

Comment:
Should we clarify here that site history is a paid feature?

Reply:
I think it's within the same doc that we are talking about Site history and have a callout that it's a paid feature. I think that for this, instead of being repetitive I'll just link to the Site History section of the same doc and have that serve as the clarification. What do you think?